server.js

```diff
@@ -61,8 +61,8 @@ app.use('/', (req, res) => {
   alt.bootstrap(JSON.stringify(res.locals.data || {}));
   const iso = new Iso();
 
-  const blogAppUrl = (req.url).indexOf('books-music-dvds/recommendations/staff-picks') !== -1;
-  const routes = blogAppUrl ? appRoutes.client : appRoutes.server;
+  const serverUrl = (req.url).indexOf('books-music-dvds/recommendations/staff-picks') !== -1;
```

Comment:
would be better to use the config.baseUrl value from a previous PR instead? less places to track if the url changes.

Reply:
FYI, I'm working on updating all the routes to use config.

Reply:
Yup, I purposely left some for @kfriedman